MapboxStatic/SnapshotOptions.swift

```diff
@@ -0,0 +1,336 @@
+     
+     If the value of this property is `nil`, a center coordinate and zoom level are automatically chosen to fit any overlays specified in the `overlays` property, and no heading or pitch is applied. If the `overlays` property is also empty, the behavior is undefined.
+     
+     The default value of this property is `nil`.
+     */
+    open var camera: SnapshotCamera?
+    
+    // MARK: Configuring the Image Output
+    
+    /**
+     The logical size of the image to output, measured in points.
+     
+     The width may not be less than 1 point or greater than 1,280 points. Likewise, the height may not be less than 1 point or greater than 1,280 points.
+     */
+    open var size: CGSize
+    
+    #if os(OSX)
+    /**
+     The scale factor of the image.
+     
+     If you multiply the logical size of the image (stored in the `size` property) by the value in this property, you get the dimensions of the image in pixels.
+     
+     The default value of this property matches the natural scale factor associated with the main screen. However, only images with a scale factor of 1.0 or 2.0 are ever returned by the Static API, so a scale factor of 1.0 of less results in a 1× (standard-resolution) image, while a scale factor greater than 1.0 results in a 2× (high-resolution or Retina) image.
+     */
+    open var scale: CGFloat = NSScreen.main()?.backingScaleFactor ?? 1
+    #elseif os(watchOS)
+    /**
+     The scale factor of the image.
+     
+     If you multiply the logical size of the image (stored in the `size` property) by the value in this property, you get the dimensions of the image in pixels.
+     
+     The default value of this property matches the natural scale factor associated with the screen. Images with a scale factor of 1.0 or 2.0 are ever returned by the Static API, so a scale factor of 1.0 of less results in a 1× (standard-resolution) image, while a scale factor greater than 1.0 results in a 2× (high-resolution or Retina) image.
+     */
+    open var scale: CGFloat = WKInterfaceDevice.current().screenScale
+    #else
+    /**
+     The scale factor of the image.
+     
+     If you multiply the logical size of the image (stored in the `size` property) by the value in this property, you get the dimensions of the image in pixels.
+     
+     The default value of this property matches the natural scale factor associated with the main screen. However, only images with a scale factor of 1.0 or 2.0 are ever returned by the Static API, so a scale factor of 1.0 of less results in a 1× (standard-resolution) image, while a scale factor greater than 1.0 results in a 2× (high-resolution or Retina) image.
+     */
+    open var scale: CGFloat = UIScreen.main.scale
+    #endif
+    
+    /**
+     A Boolean determining whether the resulting image includes the Mapbox logo.
+     
+     When shown, the Mapbox logo is located in the lower-left corner of the image. By default, this property is set to `true`.
```

Comment:
Chances are quite high that the developer is required to include the logo, so linking https://www.mapbox.com/help/attribution/ here also makes sense.